test/scripts/imgtestlib.py

```diff
@@ -63,7 +63,7 @@
 """
 
 
-def runcmd(cmd, stdin=None, extra_env=None, capture_output=True):
+def runcmd(cmd, stdin=None, extra_env=None, capture_output=True, *, cmd_root="."):
```

Comment:
Nitpick: `subprocess.run()`'s `cwd` defaults to `None`. I'd suggest defaulting to `None` here for `cmd_root`...

In addition, is there any particular reason not to allow specifying `cmd_root` as a positional argument? :thinking:

Reply:
> Nitpick: `subprocess.run()`'s `cwd` defaults to `None`. I'd suggest defaulting to `None` here for `cmd_root`...
> 

You're right.  Good call.

> In addition, is there any particular reason not to allow specifying `cmd_root` as a positional argument? 🤔

I wanted to make it forced named for `gen_manifests()` and `list_images()` and I ran with it for `runcmd()`.  I'll make it a regular arg here too, but I'll keep it named for the other two because it doesn't really _fit_ with the rest of the positional arguments, so I'd like it to stand out.

Reply:
> > In addition, is there any particular reason not to allow specifying `cmd_root` as a positional argument? 🤔
> 
> I wanted to make it forced named for `gen_manifests()` and `list_images()` and I ran with it for `runcmd()`. I'll make it a regular arg here too if I don't drop it entirely, but I'll keep it named for the other two because it doesn't really _fit_ with the rest of the positional arguments, so I wanted it to stand out.

That's fair, thanks for the explanation. I was mostly curious...